ppa/archive/views.py

```diff
@@ -91,22 +94,68 @@ def get_queryset(self, **kwargs):
             'facet.field': [field for field in self.form.facet_fields],
             # default expand sort is score desc
             'expand': 'true',
-            'expand.rows': 10,   # number of items in the collapsed group, i.e pages to display
+            'expand.rows': 2,   # number of items in the collapsed group, i.e pages to display
             'join_query': join_q,
-            'coll_query': coll_query
-            # 'rows': 50  # override solr default of 10 results; display 50 at a time for now
+            'coll_query': coll_query,
+            # enable highlighting
```

Comment:
do we need highlighting enabled here even though we're doing it using another query?

Reply:
Nope! Thanks for catching that